crates/starknet-types-core/src/curve/projective_point.rs

```diff
@@ -75,6 +79,28 @@ impl ops::AddAssign<&ProjectivePoint> for ProjectivePoint {
     }
 }
 
+impl core::ops::Neg for &ProjectivePoint {
+    type Output = ProjectivePoint;
+
+    fn neg(self) -> ProjectivePoint {
+        ProjectivePoint(self.0.neg())
+    }
+}
+
+impl core::ops::Sub<&ProjectivePoint> for &ProjectivePoint {
+    type Output = ProjectivePoint;
+
+    fn sub(self, rhs: &ProjectivePoint) -> ProjectivePoint {
+        self + &-rhs
+    }
+}
```

Comment:
can you also implement `for ProjectivePoint` plz

Reply:
Done, did the same also for the addition.